File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/util/DebugUtilsInterface.java
package de.dimedis.mobileentry.util;

public interface DebugUtilsInterface {

    String getDemoAccBarCode();

    String getDemoInitializeBarcode();

    boolean isDemoMode();

    boolean isDemoModeOn();

    String getDefaultToken();

    boolean isAdminMode();

    interface Reinit {
        void reinit();

        void next(int i);
    }

    void onTitleClick(Reinit reinit);
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/util/DebugUtilsInterface.kt
package de.dimedis.mobileentry.util

interface DebugUtilsInterface {
    fun getDemoAccBarCode(): String?
    fun getDemoInitializeBarcode(): String?
    fun isDemoMode(): Boolean
    fun isDemoModeOn(): Boolean
    fun getDefaultToken(): String?
    fun isAdminMode(): Boolean
    interface Reinit {
        fun reinit()
        fun next(i: Int)
    }

    fun onTitleClick(reinit: Reinit?)
}